docs/_templates/partials/min-version.njk

```diff
@@ -0,0 +1,15 @@
+<min-version data-tooltip="page-min-version">
+  {% if page.minver %}
+    v{{page.minver}}+
+  {% else %}
+    NEW
+  {% endif %}
+</min-version>
+
+<hx-tooltip id="page-min-version" position="right">
```

Comment:
Added a tooltip for the page-level version

Reply:
![screen shot 2018-03-19 at 7 57 27 pm](https://user-images.githubusercontent.com/545605/37629963-c67145d6-2baf-11e8-834d-1713142206a9.png)
![screen shot 2018-03-19 at 7 57 16 pm](https://user-images.githubusercontent.com/545605/37629964-c6972eb8-2baf-11e8-8cef-89a23a46c5ad.png)